### PR TITLE
Create Settings Page

### PR DESCRIPTION
**Pull request by vincentta**

### CODE REVIEW

File: app/src/main/java/com/example/musicroulette/MainActivity.java [hunk 1]
```diff
@@ -50,10 +51,26 @@ public void RequestToken() {
 
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
-        getMenuInflater().inflate(R.menu.bottom_navigation_menu, menu);
```
Comment (buitim) [inline app/src/main/java/com/example/musicroulette/MainActivity.java:52]: This seems to be the biggest issue. This commit seems fine as long as you know this is correct.

**Comment by buitim** [comment]
Just resolve the conflicts and merge @vincentta

**Review by buitim — APPROVED** [review]

File: app/src/main/res/drawable-anydpi/ic_action_discover.xml [hunk 2]
```diff
@@ -6,5 +6,5 @@
     android:tint="#FFFFFF">
     <path
         android:fillColor="#FF000000"
-        android:pathData="M6.76,4.84l-1.8,-1.79 -1.41,1.41 1.79,1.79 1.42,-1.41zM4,10.5L1,10.5v2h3v-2zM13,0.55h-2L11,3.5h2L13,0.55zM20.45,4.46l-1.41,-1.41 -1.79,1.79 1.41,1.41 1.79,-1.79zM17.24,18.16l1.79,1.8 1.41,-1.41 -1.8,-1.79 -1.4,1.4zM20,10.5v2h3v-2h-3zM12,5.5c-3.31,0 -6,2.69 -6,6s2.69,6 6,6 6,-2.69 6,-6 -2.69,-6 -6,-6zM11,22.45h2L13,19.5h-2v2.95zM3.55,18.54l1.41,1.41 1.79,-1.8 -1.41,-1.41 -1.79,1.8z"/>
+        android:pathData="M9,21c0,0.55 0.45,1 1,1h4c0.55,0 1,-0.45 1,-1v-1L9,20v1zM12,2C8.14,2 5,5.14 5,9c0,2.38 1.19,4.47 3,5.74L8,17c0,0.55 0.45,1 1,1h6c0.55,0 1,-0.45 1,-1v-2.26c1.81,-1.27 3,-3.36 3,-5.74 0,-3.86 -3.14,-7 -7,-7zM14.85,13.1l-0.85,0.6L14,16h-4v-2.3l-0.85,-0.6C7.8,12.16 7,10.63 7,9c0,-2.76 2.24,-5 5,-5s5,2.24 5,5c0,1.63 -0.8,3.16 -2.15,4.1z"/>
```
Comment (buitim) [inline app/src/main/res/drawable-anydpi/ic_action_discover.xml:9]: Choose yours instead of theirs to be safe

**Review by jimwelagu — COMMENTED** [review]
Fix the merge conflict.

**Pull request merged**